README.md

```diff
@@ -1830,6 +1831,7 @@ Options:
 -   `port` Port that Redis should be available on
 -   `parameter_group` The set of Redis parameters to use
 -   `num_nodes` Number of nodes in cache cluster
+-   `maintenance_window` Specifies the weekly time range during which maintenance on the cache cluster is performed. 
```

Comment:
I think a default value for `maintenance_window` might be useful. Also a note that the window needs to be at least 1 hour